<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsTab</class>
 <widget class="QWidget" name="OptionsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>504</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxMain">
   <item>
    <widget class="QGroupBox" name="grpDownloads">
     <property name="title">
      <string>&amp;Downloads</string>
     </property>
     <layout class="QVBoxLayout" name="vboxDownloads">
      <item>
       <widget class="QGroupBox" name="grpExtImgDownloads">
        <property name="title">
         <string>E&amp;xternal Image Downloads</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblUnmeteredConnection">
           <property name="text">
            <string>When using an unlimited
network connection:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cboUnmeteredConnection">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Don't download any images</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Download normal-resolution images</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Download high-resolution images</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblMeteredConnection">
           <property name="text">
            <string>When using a metered
network connection:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cboMeteredConnection">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Don't download any images</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Download normal-resolution images</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Download high-resolution images</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUseIntIconForSmallSizes">
        <property name="text">
         <string>Always use the internal icon (if present) for small sizes.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkStoreFileOriginInfo">
        <property name="text">
         <string>Store cached file origin information using extended attributes.
This helps to identify where cached files were downloaded from.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hboxGameTDBPAL">
        <item>
         <widget class="QLabel" name="lblGameTDBPAL">
          <property name="text">
           <string>Language for PAL titles on GameTDB:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LanguageComboBox" name="cboGameTDBPAL"/>
        </item>
        <item>
         <spacer name="hspcGameTDBPAL">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpOptions">
     <property name="title">
      <string>&amp;Options</string>
     </property>
     <layout class="QVBoxLayout" name="vboxOptions">
      <item>
       <widget class="QCheckBox" name="chkShowDangerousPermissionsOverlayIcon">
        <property name="text">
         <string>Show a security overlay icon for ROM images with
&quot;dangerous&quot; permissions.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEnableThumbnailOnNetworkFS">
        <property name="text">
         <string>Enable thumbnailing and metadata extraction on network
file systems. This may slow down file browsing.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkThumbnailDirectoryPackages">
        <property name="text">
         <string>Enable thumbnailing and metadata extraction of directory-based
packages, e.g. for Wii U. This may slow down file browsing.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowXAttrView">
        <property name="text">
         <string>Show the Extended Attributes tab.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vspcDownloads">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>197</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LanguageComboBox</class>
   <extends>QComboBox</extends>
   <header>LanguageComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkUseIntIconForSmallSizes</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkShowDangerousPermissionsOverlayIcon</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEnableThumbnailOnNetworkFS</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkStoreFileOriginInfo</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboGameTDBPAL</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboUnmeteredConnection</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboMeteredConnection</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grpExtImgDownloads</sender>
   <signal>clicked(bool)</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkShowXAttrView</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkThumbnailDirectoryPackages</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>modified()</signal>
  <slot>reset()</slot>
  <slot>loadDefaults()</slot>
  <slot>save()</slot>
  <slot>optionChanged_slot()</slot>
 </slots>
</ui>
